docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,348 @@
+# Setting up a microk8s cluster
+
+Follow the steps below to set up a microk8s cluster from scratch, including all the necessary dependencies and configuration.
+
+## 1. Concepts
+
+### Kubeconfig
+
+Any tool that operates on top of a cluster needs to know which cluster it is going to work on. This execution context is typically called the `kubeconfig` which is stored in the `~/.kube/config` file. This filed basically defines a list of clusters and it holds several pieces of information per cluster entry: the address of the cluster (e.g. https://104.248.34.242:164309), its user friendly name (e.g. microk8s-cluster). It finally holds a reference to the current context. This is the active cluster against which any kubernetes control (`kubectl`) command you run will get executed.
+
+### Helm and tiller
+
+Two executables will get installed onto the cluster as part of this process: `helm` and `tiller`. The former is responsible for resolving the configuration of the chart to be installed, while issuing the correct install commands onto the cluster. The latter is responsible for receiving the install commands issued by `helm`, as well as managing the lifecycle of the components that have been installed. `helm` is the client facing side, while `tiller` is the server/cluster facing side.
+
+## 2. Prepare your environment
+
+Prepare your environment to set up the microk8s cluster:
+
+1. a machine running Ubuntu 18.04 LTS version.
+2. establish a remote session (ssh) onto this machine.
+
+Assuming that you are starting from a blank slate, the first step is to install microk8s. Otherwise, please jump on to the [Configuring microk8s](###Configuring-microk8s) section.
```

Comment:
```suggestion
Assuming that you are starting from a blank slate, the first step is to install microk8s. Otherwise, please jump on to the section [Configuring microk8s](###4.-Configuring-microk8s).
```

Reply:
Make sure the internal links are working on the final version.